bug-bounty-list/bug-bounty-list.json

```diff
@@ -6008,6 +6008,16 @@
     "swag": false,
     "hall_of_fame": true,
     "safe_harbor": "partial"
+  },
+   {
+    "program_name": "Samsung",
```

Comment:
Is this just Samsung Mobile? we should probably be specific

Reply:
yes, sorry should have specified. Thanks